.github/workflows/central-publish.yml

```diff
@@ -0,0 +1,29 @@
+name: Publish to the Ballerina central
+
+on:
+    workflow_dispatch:
+
+jobs:
+    publish-release:
+        runs-on: ubuntu-latest
+        if: github.repository_owner == 'ballerina-platform'
+        steps:
+            -   uses: actions/checkout@v2
+            -   name: Set up JDK 11
+                uses: actions/setup-java@v1
+                with:
+                    java-version: 11
+            -   name: Set version env variable
+                run: echo "VERSION=$((grep -w 'version' | cut -d= -f2) < gradle.properties | rev | cut --complement -d- -f1 | rev)" >> $GITHUB_ENV
```

Comment:
I think we can remove this. Otherwise when we are trying to push from an already released version, it will remove the `alpha` part from the version string.

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/123/commits/7dfa139ff07a020cff07f7e4d976b0363e7271b3